websites/api-docs-smoke-test/src/generated-api-specs/test/api.raml

```diff
@@ -0,0 +1,111 @@
+#%RAML 1.0
```

Comment:
Are those changes still necessary? I thought you fixed that already...

Reply:
It must be the merge with the master, as I did not add these files in this branch.

Reply:
Can you remove those things then? If they are here it means that something went wrong with updating your branch.